src/sass/components/_post-box.scss

```diff
@@ -51,4 +63,26 @@
       padding-top: 30px;
     }
   }
+
+}
+
+.post-box {
+
+  &:hover {
+    text-decoration: none;
+
+    h5 {
+      color: #9b6520;
+    }
+
+    .details {
+      color: $primary;
+    }
+    .btn-main-more {
+      color: #ffffff !important;
```

Comment:
Please don't use `!important` - it's very, very, very bad practice… BTW you don't need to override color to `#ffffff`, because `.btn-main-more` has it by default.

Reply:
Fixed in new commit